app-backend/api/src/main/scala/tasks/Routes.scala

```diff
@@ -57,6 +57,10 @@ trait TaskRoutes
           }
         }
       }
+    } ~ pathPrefix("random-session") {
```

Comment:
I think to get this to match the endpoint in the issue would just be this change and updating the integration test too. I don't have super-strong feelings on this, so I don't think it's a requirement to change this to merge, but I think unless there's a good reason to switch it, defaulting to the endpoint in the issue is a good idea.

```suggestion
    } ~ pathPrefix("session") {
```